geomesa-hbase/geomesa-hbase-datastore/src/main/java/org/locationtech/geomesa/hbase/filters/JSimpleFeatureFilter.java

```diff
@@ -267,7 +267,7 @@ private static int getLen(String s) {
     }
 
     private static JSimpleFeatureFilter.Transformer buildTransformer(String transformString, String transformSchemaString) {
-        if(Strings.isEmpty(transformString)) {
+        if(StringUtils.isEmpty(transformString)) {
```

Comment:
we could probably just change this to `transformString == null || transformString.isEmpty` and avoid the parboiled import.